### PR TITLE
[DOCS-263] Reports

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Pentest Reports | https://deploy-preview-228--cobalt-docs.netlify.app/platform-deep-dive/pentests/reports/ | Minor updates, added shortcodes |
| What's in a Pentest Report | https://deploy-preview-228--cobalt-docs.netlify.app/platform-deep-dive/pentests/reports/report-contents/ | Minor updates |
| Glossary | https://deploy-preview-228--cobalt-docs.netlify.app/getting-started/glossary/ | Added report types definitions |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Pentests/Reports/_index.md [hunk 1]
```diff
@@ -12,16 +12,23 @@ Download a pentest report to view security issues that our pentesters found.
...
-| Full Report | Comprehensive Pentest | Includes the following report sections, beyond attestation:<br>  <br>- Executive Summary <br>- Scope of Work <br>- Methodology <br>- Summary of Findings <br>- Recommendations <br>- Post-Test Remediation     | Yes |
-| Full Report + Finding Details | Comprehensive Pentest | Adds details of every test finding to the full report. Details include:<br>  <br>- Vulnerability Type <br>- Description <br>- Proof of Concept <br>- Severity <br>- Suggested Fix     | Yes |
+| Automated Report | Agile Pentest | {{% automated-report %}} Includes the following sections:<br><br>- Pentester user information<br>- Executive Summary<br>- Methodology<br>- Post-Test Remediation<br>- Finding Details | No |
+| Customer Letter | Comprehensive Pentest | {{% customer-letter %}} Includes:<br>  <br>- Executive Summary <br>- Methodology | Yes |
+| Attestation Report | Comprehensive Pentest | {{% attestation-report %}}<br><br>Adds the following information to the Customer Letter: <br> <br>- Pentester user information <br>- An overall list of findings  | Yes |
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Reports/_index.md:28]: @TheInstaGrahame Hi Grahame, I need your help with reviewing some content related to reports.

On this [page](https://developer.cobalt.io/platform-deep-dive/pentests/reports/), we state the an Attestation Report includes the following:
- Pentester user information 
- An overall list of findings (I can't see it in the app)

What is the real difference between a Customer Letter and Attestation Report?

I added definitions for all report types to `layouts/shortcodes`. Can you please review? Thank you.

File: content/en/Getting started/glossary.md [hunk 2]
```diff
@@ -78,11 +78,15 @@ an individual, a group, or even a nation-state. Specified as "attacker" in Cobal
 
 ## Attestation Letter
 
-{{% attestation-letter %}}
+{{% attestation-letter %}} You cannot customize an Attestation Letter. {{% learn-more-reports %}}
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/glossary.md:81]: @mjang-cobalt FYI. There is a plan to make Attestation Letters customizable.

Comment (mjang-cobalt) [reply content/en/Getting started/glossary.md:81]: I remember reading that, thanks for highlighting.

File: content/en/Platform Deep Dive/Pentests/Reports/_index.md (hunk 1 above)
Comment (TheInstaGrahame) [reply content/en/Platform Deep Dive/Pentests/Reports/_index.md:28]: Hey!

So, for context, the Attestation Report is what we had before we invented the Attestation Letter. THe request we had from customers was a one-page letter indicating that we had performed a test. 

That said, the Attestation Report includes: 
- Executive Summary
- Executive Analysis + Finding Breakdown
- Methodology/Rating Info

The Customer Letter has:
- Executive Summary
- Methodology/Rating Info

And just to round out the trio, Attestation Letter has:
- Executive Summary

The Executive Summary section of the report is essentially two paragraphs that say "Cobalt came and performed X testing. We had Y testers, and did it from A Date to B Date."

The Analysis includes info about the test itself, at a high level.

The methodology and rating info is boilerplate, and mostly serves to explain our rating system.

(FYI: Product is currently working on an update that will put the Finding Breakdown table into the Attestation Letter, and make it an option for customers to show or hide. That's not quite done yet, but I'll keep you and Mike posted on it)

Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Pentests/Reports/_index.md]: Thank you @TheInstaGrahame, I made changes based on our discussion in Slack.

**Comment by ana-dashuk-cobalt** [comment]
@mjang-cobalt This PR is ready for your review.

**Review by mjang-cobalt — APPROVED** [review]
Good stuff, and I thank @TheInstaGrahame for his guidance. @ana-dashuk-cobalt you're welcome to merge.

**Pull request merged**